labellab-client/src/components/dashboard/css/previous.css

```diff
@@ -1,3 +1,10 @@
 .previous-card-image{
     width: 100% !important;
+}
+
+.card-headers .header {
+    width: 100%;
+    overflow: hidden;
+    text-overflow: ellipsis; 
```

Comment:
Instead of giving ellipsis you can break the word or line.

Reply:
@DonOmalVindula

Reply:
> Instead of giving ellipsis you can break the word or line.

I will do the changes and commit. Sorry about the delay.

Reply:
I have made the changes @ivantha @Tushar19varshney. please check